client/src/main/java/io/pravega/client/connection/impl/FlowHandler.java

```diff
@@ -185,18 +147,6 @@ public void process(Reply cmd) {
             return;
         }
 
-        if (cmd instanceof WireCommands.DataAppended) {
-            final WireCommands.DataAppended dataAppended = (WireCommands.DataAppended) cmd;
-            final AppendBatchSizeTracker batchSizeTracker = getAppendBatchSizeTracker(dataAppended.getRequestId());
-            if (batchSizeTracker != null) {
-                long pendingAckCount = batchSizeTracker.recordAck(dataAppended.getEventNumber());
-                // Only publish client side metrics when there is some metrics notifier configured for efficiency.
-                if (!metricNotifier.equals(MetricNotifier.NO_OP_METRIC_NOTIFIER)) {
-                    metricNotifier.updateSuccessMetric(CLIENT_OUTSTANDING_APPEND_COUNT, writerTags(dataAppended.getWriterId().toString()),
-                            pendingAckCount);
-                }
```

Comment:
Are we loosing this metric?

Reply:
We have never used it since it was introduced as far as I know, and it may reduce the computation overhead of the client when processing the ack. We can bring it back in the future if necessary.